app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,24 @@
+class NotificationsController < ApplicationController
+
+
+  def index
+    @notifications=current_user.notifications.order("created_at DESC")
+
+  end
+
+  def show
+    @notification=current_user.notifications.find(params[:id])
+  end
+
+  def destroy
+    @notification=Notification.find_by_id(params[:id])
```

Comment:
There is no write destroy method for notification. 
Write a cron job that will destroy notification that are older than 30 days.

Reply:
I saw notification remove option on Facebook therefore I implemented it.

Reply:
Okay after I will see your demo. I will let you know what to do with this.